app/views/devise/registrations/edit.html.erb

```diff
@@ -5,55 +5,56 @@
       <%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
       <%= devise_error_messages! %>
       <div class="field">
-       <div><%= f.label :name %><br />
-        <%= f.text_field :name, autofocus: true ,class: 'form-control margin-bottom'%></div>
+      <div><%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true ,class: 'form-control margin-bottom'%>
       </div>
-            
+      </div>
+      
       <div class="field">
         <div><%= f.check_box :is_female, {}, true %> <%= f.label :is_female, "Female" %> </div>
```

Comment:
i hope this is_female part is being fixed.

Reply:
Yes, have fixed.